go.mod

```diff
@@ -12,6 +12,7 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gorilla/websocket v1.5.0
```

Comment:
Did you correctly vendored your dependencies? This should be in the previous `require` block as it a direct dependency.